install/linux.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_label: Linux
+---
+
+# Installing RGBDS on Linux
+
+## Using your package manager
+
+First, please check if RGBDS is available in your distribution's software center or package manager repositories.
+If a satisfactory version is available, it is a good idea to install from there.
+
+## Using our pre-built binaries
```

Comment:
Standardize on "binary or "executable" terminology?

Reply:
Agree

Reply:
Let's use "executables" then, as these are probably more meaningful to end users.